var/spack/repos/builtin/packages/ntl/package.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ntl(Package):
+    """
+    NTL  -- a library for doing numbery theory --  version 11.5.1
+    Release date: 2021.06.23
+
+    Author: Victor Shoup (victor@shoup.net)
```

Comment:
Would it be possible to use:
```suggestion
    NTL  -- a library for doing numbery theory
```
? The information removed is subject to more frequent changes and should be found in the homepage of the package that is reported below.